GitHubCodespaces.ps1

```diff
@@ -769,11 +920,22 @@ filter Stop-GitHubCodespace
         GitHub Apps must have write access to the codespaces_lifecycle_admin repository permission to use this endpoint.
 #>
     [CmdletBinding(
+        DefaultParameterSetName = 'AuthenticatedUser',
```

Comment:
That is deliberate and prevents an error if no default is specified.

Reply:
I made CodespaceName an optional param of the AuthenticatedUser parameter set for readability.

Hopefully, this clarifies that if no parameters are specified, it will [list-codespaces-for-the-authenticated-user](https://docs.github.com/en/rest/codespaces/codespaces?apiVersion=2022-11-28#list-codespaces-for-the-authenticated-user).  If CodespaceName is specified, it will [get-a-codespace-for-the-authenticated-user](https://docs.github.com/en/rest/codespaces/codespaces?apiVersion=2022-11-28#get-a-codespace-for-the-authenticated-user).

CodespaceName is not applicable to the Organization parameter set.